compose-builder/add-taf-app-services.yml

```diff
@@ -0,0 +1,50 @@
+version: '3.7'
+
+services:
+  app-service-blackbox-tests:
+    image: ${REPOSITORY}/docker-app-service-configurable${ARCH}:${APP_SERVICE_VERSION}
+    ports:
+      - 48105:48105/tcp
+    container_name: app-service-blackbox-tests
+    hostname: app-service-blackbox-tests
+    env_file:
+      - common.env
+    environment:
+      EDGEX_PROFILE: blackbox-tests
+      SERVICE_HOST: app-service-blackbox-tests
+      SERVICE_PORT: 48105
+    depends_on:
+      - consul
+      - data
+    read_only: true
+    networks:
+      - edgex-network
+    security_opt:
+      - no-new-privileges:true
+
+  scalability-test-mqtt-export:
+    image: ${REPOSITORY}/docker-app-service-configurable${ARCH}:${APP_SERVICE_VERSION}
+    command: "-s -cp=consul.http://edgex-core-consul:8500 --registry --confdir=/res"
```

Comment:
Just FYI and note to myself, the `-cp=...` part can be replaced with the global env. defined in my draft PR.  I'll do a search for this and replace it so that there is only one place to modify in future if anything changed

Reply:
@lenny-intel Should this line also be removed?

Reply:
@jim-wang-intel , as @cherrycl pointed out, I missed removing this one. Fix!